vendor/engines/saml_authentication/app/controllers/saml_authentication/sessions_controller.rb

```diff
@@ -2,13 +2,69 @@ module SamlAuthentication
 
   class SessionsController < Devise::SamlSessionsController
 
+    # Some SSO providers are strict and require that the name_identifier matches
+    # what's logged in on their system for SLO.
+    before_action :store_logging_out_username, only: :destroy
+    after_action :store_winning_strategy, only: :create
+
+    # Overrides SamlSessionsController because we don't want to use saml_session_index_key
+    # because it logs you out of other browsers/sessions when you log in on another
```

Comment:
Can you clarify in the docs exactly what the behavior is with the respect to this override? Can you have more than one session with the IdP going at the same time, i.e. I'm signed in to email and/or NUcore on computer A, but signed into to email and NUcore on computer B, so that when I sign out of NUcore with SLO I am signed out of both email and NUcore on B, but my sessions on A are not affected?